index.js

```diff
@@ -2,32 +2,41 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var loaderUtils = require("loader-utils"),
-	path = require("path");
-module.exports = function() {};
-module.exports.pitch = function(remainingRequest) {
-	if(this.cacheable) this.cacheable();
-	var query = loaderUtils.getOptions(this) || {};
+
+var path = require("path");
+
+var loaderUtils = require("loader-utils");
+var validateOptions = require('schema-utils');
+
+module.exports = function () {};
+
+module.exports.pitch = function (request) {
+	if (this.cacheable) this.cacheable();
+
+	var options = loaderUtils.getOptions(this) || {};
```

Comment:
Check for sure. If it doesn't, figure out why it doesn't return an `{}`. That would seem like a sensible default or else we end up with the check everywhere.